src/test/java/org/jenkinsci/plugins/tuleap_git_branch_source/jcasc/JCAsCTest.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.plugins.tuleap_git_branch_source.jcasc;
+
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import hudson.security.ACL;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import org.jenkinsci.plugins.tuleap_credentials.TuleapAccessToken;
+import org.jenkinsci.plugins.tuleap_git_branch_source.config.TuleapConfiguration;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.*;
```

Comment:
Is it possible to import "org.hamcrest.MatcherAssert.assertThat;" instead ? 
assertThat from junit is deprecated and it's recommended to use the assertThat from hamcrest*

*https://junit.org/junit4/javadoc/latest/deprecated-list.html#method

Reply:
didn't know, thanks!